lib/main.js

```diff
@@ -61,7 +60,9 @@ export function consumeStatusBar (statusBar) {
     statusBarItem.setLineEndings(lineEndings)
   }
 
-  let debouncedUpdateTile = _.debounce(updateTile, 0)
```

Comment:
Yeah, the idea behind using `debounce` here is to minimize the number of times we loop through every line in the buffer to determine the overall line ending style. Using `setTimeout` does not achieve this.